content/get-started/quickstart/index.md

```diff
@@ -27,3 +27,4 @@ redirect_from:
   - /github/getting-started-with-github/quickstart
 ---
 
+Bb
```

Comment:
Re: [github/docs] I have a question regarding the repository search API (I hope this is the right place, otherwise please tell me where I can ask about this): [https://docs.github.com/en/rest/reference/search#search-repositories](https://docs.github.com/en/rest/reference/search#search-repositories) (Issue #16901)